app/point.py

```diff
@@ -12,7 +12,6 @@ def __eq__(self, other: Any) -> bool:
         return self.x == other.x and self.y == other.y
 
     def __hash__(self) -> int:
-        # Change the implementation of the hash to debug your code.
```

Comment:
Do not make changes in the `app/point.py`, this file should be absent in the PR.